app/templates/pdf/ticket_purchaser.html

```diff
@@ -1,99 +1,164 @@
+                    #%(invoice_number)s',invoice_number="")}}{{order.get_invoice_number()}}</span></td>
+        </tr>
+    </table>
+    <div class="main_content">
+        <table>
+            <tr>
+                <td id="event_name" colspan=2>
+                    <span>{{order.event.name}}</span><br>
+                    <span id="role">{{_('Guest')}}</span><br>
+                    <span id="location">
+                        {% if order.event.online %}
+                        {% if order.event.location_name %}
+                        <span>{{_('Event taking place online and at
+                            %(location)s',location="")}}{{order.event.location_name}}</span>
+                        {% else %}
+                        <span>{{_('Event taking place online')}}</span>
+                        {% endif %}
+                        {% elif order.event.location_name %}
+                        <span>{{_('Event taking place at
+                            %(location)s',location="")}}{{order.event.location_name}}</span>
+                        {% else %}
+                        <span>{{_('Location or online event details to be announced')}}</span>
+                        {% endif %}
+                    </span>
+                    <br>
+                    <span>
+                        {% if order.event.starts_at %}
+                        <span class="time">{{_('From: %(start_date)s',start_date="")}}{{order.event.starts_at |
+                            datetime(order.event.timezone)}}</span>
+                        {% endif %}
+                        <br>
+                        {% if order.event.ends_at %}
+                        <span class="time">{{_('To: %(end_date)s',end_date="")}}{{order.event.ends_at |
+                            datetime(order.event.timezone)}}</span>
+                        {% endif %}
+                    </span>
+                    <span>
+                        <table id="order_details">
+                            <tr>
+                                <td>
+                                    <span class="custom_label">{{_('Order Information')}}</span>
+                                    <p class="data">{{_('Order
+                                        %(invoice_number)s',invoice_number="")}}{{order.get_invoice_number()}}</p>
+                                </td>
+                                <td>
+                                    <span class="custom_label">{{_('Name')}}</span>
+                                    <p class="data">{{holder.name}}</p>
+                                </td>
+                            </tr>
+                        </table>
+                    </span>
+                </td>
+                <td align="center">
+                    <!-- <img src="{{order.event.original_image_url}}" alt=""><br><br><br> -->
+                    <img src="https://miun.imagevault.media/publishedmedia/co81r635gbhgo2ahfuci/Yran-4start.webp" alt=""
```

Comment:
revert to original image url

Reply:
Ok can you please tell what should I keep in the src is the one commented correct or should I make any change?